tests/BoardTest.elm

```diff
@@ -0,0 +1,30 @@
+module BoardTest exposing (..)
+
+import Test exposing (..)
+import Board exposing (..)
+import Expect
+import List exposing (isEmpty, length)
+
+
+all : Test
+all =
+    describe "Board Test"
+        [ test "A new board is empty" <|
+            \() ->
+              Board.new 0
```

Comment:
Good point, I should replace this with a test for possible moves or something. I.e. the thing is empty of marks.